Documentation/HowToGuides/PassingAFloatValueFromAnInteractorToAnInteractable/README.md

```diff
@@ -0,0 +1,190 @@
+
+> The property will be 0 by default so change it to 2
+
+![Change Size Element To 2](assets/images/ChangeSizeElementTo2.png)
+
+### Step 11
+
+Add the 2 `Interactions.ActionPublisher` GameObjects we created for the `LeftControllerAlias` and the `RightControllerAlias` to the newly created Elements.
+
+Drag and drop the `CameraRigs.TrackedAlias -> Aliases -> LeftControllerAlias -> Interactions.ActionPublisher` GameObject from the Unity hierachy window into the newly displayed Element `0` field within the `Source Publisher Elements` parameter on the `Interactable Action Receiver Facade`. Make sure to do the same for the `RightControllerAlias`.
+
+![Drag And Drop Action Publishers](assets/images/DragAndDropActionPublishers.png)
+
+### Step 12
+
+Expand the `OpacityChangingInteractable -> Interactions.ActionReceiver` GameObject and then expand the `OutputActions` GameObject, you'll see 3 action GameObjects.
+
+* `Boolean Action` - This will be used if the input action from our controller is a boolean e.g. a button.
+* `Float Action` - This will be used if the input action from our controller is 1D Axis e.g. a trigger press.
+* `Vector2 Action` - This will be used if the input action from our controller is like a trackpad movement.
+
+In this example we are going to use the `Float Action` because we are going to use a trigger press.
+
+### Step 13
+
+We'll need to create a small script that allows us to change the opacity of our interactable GameObject to test that the trigger value from the controller passes over to the interactable.
+
+Create a new script by selecting `Main Menu -> Assets -> Create -> C# Script` in the Unity software and name it `OpacityChanger`.
+
+Copy and paste the below code into the newly created `OpacityChanger` script:
+
+```
+using UnityEngine;
+
+public class OpacityChanger : MonoBehaviour
+{
+    public GameObject meshToChange;
+
+    protected MeshRenderer materialRenderer;
+
+    protected void OnEnable()
+    {
+        materialRenderer = meshToChange.GetComponentInChildren<MeshRenderer>();
+    }
+
+    public void ChangeOpacity(float value)
+    {
+        Color currentColor = materialRenderer.material.color;
+        currentColor.a = 1f - value;
+        materialRenderer.material.color = currentColor;
+    }
+}
+```
+
+This simple script has a single method called `ChangeOpacity` which will take a float value that sets the opacity of the material that is used on the GameObject.
+
+Create a new GameObject as a child of the `OpacityChangingInteractable` and name it `OpacityChanger`. Then add the new `OpacityChanger` component to the GameObject.
+
+### Step 14
+
+We need to tell our `OpacityChanger` component what mesh to change. Drag and Drop the `Interactions.Interactable` GameObject into the `Mesh to Change` property on the `OpacityChanger` component.
+
+![Opacity Changer Script](assets/images/OpacityChangerScript.png)
```

Comment:
Show the arrow dragging the interactable into the mesh to change property, and make sure the mesh to change property actually shows the interactable in there instead of saying `None (GameObject)`

![image](https://user-images.githubusercontent.com/1029673/74438609-7a124280-4e62-11ea-94ce-cb76e8f83ea6.png)